### PR TITLE
Support custom chromedriver CDN url

### PR DESCRIPTION
**Pull request by fengmk2**
It will very helpful for users from China.

### CODE REVIEW

**Comment by xudafeng** [comment]
+1

File: lib/install.js [hunk 1]
```diff
@@ -10,7 +10,8 @@ import { exists } from './utils';
 const { system, tempDir } = support;
 
 const CD_VER = "2.16";
-const CD_CDN = "http://chromedriver.storage.googleapis.com";
+const CD_CDN = process.env.npm_config_chromedriver_cdnurl || process.env.CHROMEDRIVER_CDNURL ||
```
Comment (jlipps) [inline lib/install.js]: please break this over 3 lines:

```js
const CD_CDN = process.env.npm_config_chromedriver_cdnurl ||
               process.env.CHROMEDRIVER_CDNURL ||
               "http://chromedriver.storage.googleapis.com";
```

**Comment by jlipps** [comment]
one comment, otherwise looks great!

File: lib/install.js [hunk 2]
```diff
@@ -10,7 +10,9 @@ import { exists } from './utils';
 const { system, tempDir } = support;
 
 const CD_VER = "2.16";
-const CD_CDN = "http://chromedriver.storage.googleapis.com";
+const CD_CDN = process.env.npm_config_chromedriver_cdnurl ||
+               process.env.CHROMEDRIVER_CDNURL ||
+               "http://chromedriver.storage.googleapis.com";
 const CD_BASE_DIR = path.resolve(__dirname, "..", "..", "chromedriver");
```
Comment (fengmk2) [inline lib/install.js]: @jlipps done

**Comment by jlipps** [comment]
great thanks!

**Comment by fengmk2** [comment]
Happy to see this merged. Can you publish a patch version for us to use it now?

**Comment by jlipps** [comment]
a patch version has been released!

**Pull request merged**